components/network/index.ts

```diff
@@ -102,7 +118,7 @@ export class VisNodes extends Vis.DataSet<VisNode> {
   }
 }
 
-export class VisEdges extends Vis.DataSet<VisEdge> {
+export class VisEdges extends DataSet<VisEdge> {
```

Comment:
As with nodes either `DataSet<VisEdge, 'id'>` or `DataSetEdges`.

Reply:
thanks, fixed that